ipaserver/install/cainstance.py

```diff
@@ -433,26 +443,14 @@ def configure_instance(self, host_name, dm_password, admin_password,
                 else:
                     self.step("importing RA certificate from PKCS #12 file",
                               self.__import_ra_cert)
-
             if not ra_only:
                 if not self.clone:
                     self.step("publishing the CA certificate",
                               self.__export_ca_chain)
                     self.step("adding RA agent as a trusted user", self.__create_ca_agent)
-                self.step("authorizing RA to modify profiles", configure_profiles_acl)
-                self.step("authorizing RA to manage lightweight CAs",
-                          configure_lightweight_ca_acls)
-                self.step("Ensure lightweight CAs container exists",
-                          ensure_lightweight_cas_container)
-                if self.clone and not promote:
-                    self.step(
-                        "Ensuring backward compatibility",
-                        self.__dogtag10_migration)
                 self.step("configure certificate renewals", self.configure_renewal)
                 self.step("Configure HTTP to proxy connections",
                           self.http_proxy)
-                # This restart is needed for ACL reload in CA, do not remove it
```

Comment:
This restart is to avoid a CVE. It looks like a restart happens later but I didn't spend timing finding out exactly where. I don't know if this was simply to ensure a restart or if it had to be done at this point.

Reply:
Not quite right - this comment was added during the secure-ajp-connection commit, but only to explain why this restart was needed.  The restart already occurred here for reason stated in the comment.  The restart was not introduced during and is not needed for the secure-ajp-connection change because a restart already happens after `self.secure_ajp_connection` gets executed.

So yes the restart was needed, no it was not for the CVE. But with this change, the ACL tweaks are now being performed before the server is restarted.  So it is OK to remove this restart.

@tiran please add a comment near steps that tweak the Dogtag ACLs, to explain that the server must be (re)started after those steps.